<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cliente</class>
 <widget class="QDialog" name="Cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Cliente</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>871</width>
     <height>621</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Contato do Cliente</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>59</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E-mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_email">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>521</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fone">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_fechar">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_gravar">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Gravar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>59</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Celular</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nome">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>521</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>59</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>59</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Fone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_celular">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_clientes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>771</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Entre com nome ou telefone</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_procurar">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>170</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Procurar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_procura">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>671</width>
       <height>22</height>
      </rect>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Endereço</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>841</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string>Endereços Cadastrados</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_bloco">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>190</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numapto">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>190</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_endcadast">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>711</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_numero">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>170</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Número</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cep">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>220</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>[CEP]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_bairro">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>[Bairro]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_estado">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>[Estado]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>170</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Bloco</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cidade">
      <property name="geometry">
       <rect>
        <x>278</x>
        <y>220</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>[Cidade]</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rua">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>381</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numero">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>170</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Apto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_limpar">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>220</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_apagar">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>30</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Apagar
Endereços</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_gravar_2">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>190</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Gravar</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>841</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Inserir novo endereço</string>
     </property>
     <widget class="QTableView" name="tableView_enderecos">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>811</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_cidade">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Cidade</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Estado</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_procurar_2">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>80</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Procurar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_estados">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_procurarua">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>691</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Entre com bairro ou rua&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
